doc/usage/extensions/math.rst

```diff
@@ -105,8 +105,9 @@ built:
    Unfortunately, this only works when the `preview-latex package`_ is
    installed. Therefore, the default for this option is ``False``.
 
-   Currently this option is only used when ``imgmath_image_format`` is
-   ``'png'``.
+   .. versionchanged:: 2.1.0
```

Comment:
I've written `versionchnaged:: 2.1` (not 2.1.0) in another PR. I don't know what is best.

Reply:
I will use `2.1` too. I still need to adjust a bit to our new branch/release model.